gear/gear/auth.py

```diff
@@ -12,19 +12,34 @@
 from hailtop.auth import async_get_userinfo
 from hailtop.config import get_deploy_config
 
+from .time_limited_max_size_cache import TimeLimitedMaxSizeCache
+
 log = logging.getLogger('gear.auth')
 
 deploy_config = get_deploy_config()
 
 BEARER = 'Bearer '
 
+TEN_SECONDS_NANOSECONDS = int(1e10)
```

Comment:
Should this just be `TEN_NANOSECONDS`?

Reply:
It's 10 seconds worth of nanoseconds, not just 10 nanoseconds. Should I name it `TEN_SECONDS_IN_NANOSECONDS` or maybe `TEN_SECONDS_IN_NS`

Reply:
I like `TEN_SECONDS_IN_NANOSECONDS`.